src/services/api/call.tsx

```diff
@@ -8,15 +8,22 @@ import {
   RecordExistsError,
 } from '../errors';
 
+export interface RequestCallOptions {
+  /** always return a list for succesful requests */
+  forceListReturn?: boolean;
+}
+
 /**
  * Sends request to server, appending all global headers and handling responses and errors.
- * @param {string} endpoint - URL endpoint
- * @param {Object} init - Request properties.
- * @param {Object} requestOptions - options to be passed to the Request contstructor
- * @param {object} callOptions - other options
- * @param {object} callOptions.forceListReturn - always return a list for succesful requests
  */
-async function request(endpoint, requestOptions, callOptions) {
+async function request(
+  /** URL endpoint */
```

Comment:
Since this is a function definition it would be better to put these arg comments in the docstring to match the other ways function properties are currently documented

Reply:
[5d3f6e1](https://github.com/bcgsc/pori_graphkb_client/pull/52/commits/5d3f6e1f522c66bb848c8756cccaeadfa50f61f9)